client/components/Events/EventHistory.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import * as actions from '../../actions';
+import * as selectors from '../../selectors';
+import {getItemInArrayById} from '../../utils';
+import {get, includes} from 'lodash';
+import {AbsoluteDate} from '../index';
+
+export class EventHistoryComponent extends React.Component {
+    componentWillMount() {
+        const {item, fetchEventHistory} = this.props;
+
+        if (get(item, '_id', null) !== null) {
+            fetchEventHistory(item._id);
+        }
+    }
+
+    componentWillReceiveProps(nextProps) {
+        // If the Assignment item has changed, then load the new history
+        const nextId = get(nextProps, 'item._id', null);
+        const currentId = get(this.props, 'item._id', null);
+
+        if (nextId !== currentId || get(nextProps, 'historyItems.length') !== get(this.props, 'historyItems.length')) {
+            this.props.fetchEventHistory(nextId);
+        }
+    }
+
+    closeAndOpenDuplicate(duplicateId) {
+        // this.props.closeEventHistory();
+        this.props.openEventPreview(duplicateId);
+    }
+
+    render() {
+        const {users} = this.props;
+        const displayUser = (recievedUserId) => get(getItemInArrayById(users, recievedUserId), 'display_name');
+
+        return (
+            <div>
+                <ul className="history-list">
```

Comment:
`<ul>` should be the parent component (removing the parent div node, as it is not required).

Also the `history-list` class is adding `padding: 10px` which is not required since `side-panel__content-block` is providing the padding for us